test/src/main/java/org/apache/accumulo/test/ImportExportIT.java

```diff
@@ -92,15 +92,20 @@ public void testExportImportThenScan() throws Exception {
       FileSystem fs = cluster.getFileSystem();
       Path tmp = cluster.getTemporaryPath();
       log.info("Using FileSystem: " + fs);
-      Path baseDir = new Path(tmp, getClass().getName());
+      // Add Fs.defaultFS to baseDir so importDirectory has the full path for Standalone Instance
+      String defaultFS = fs.getConf().get(FileSystem.FS_DEFAULT_NAME_KEY);
```

Comment:
I am not completely sure, but I think this could possibly be replaced with `fs.getUri().toString()`.

Reply:
Actually, if that works then could do `Path baseDir = new Path(fs.getUri() + tmp, getClass().getName());`

Reply:
Yeah, you're right. That does work. I could have sworn I tried that already but I guess I was mistaken. Thanks for the help. I will make the changes to the rest of them.